homework01/vector.h

```diff
@@ -1,5 +1,35 @@
 
 class Vector
 {
-	// TODO
+ public:
+  Vector(double x = 0., double y = 0.)
```

Comment:
I would not use default values here: it seems wrong (and not so useful, as a feature) to construct a 2D Vector with only `x`. It's better to be explicit and have 2 constructors. And if you do this change, you could use delegating constructors, a C++11 feature: https://en.cppreference.com/w/cpp/language/initializer_list#Delegating_constructor